packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -1085,7 +1103,14 @@ export const Table = <
             </Box>
           )}
           {isLoading && data.length !== 0 && (
-            <Box className='iui-table-row'>
+            <Box
+              className='iui-table-row'
+              // To center the row even when scrolling horizontally
+              style={{
+                position: 'sticky',
```

Comment:
what specific case does this handle?

inline styles should generally be avoided, so i'm trying to think of ways we can move this into the CSS.

Reply:
I did this to handle the case where the table is loading when there is pre-existing data.

I thought of adding these styles inline because I noticed that it was also done on [L1116](https://github.com/iTwin/iTwinUI/pull/1725/files/f1aadc6ad7a4578ac17f3ac95cc9ace86224eb4e#diff-ef4196b6638c028042adefbb785a48b755d08db5caa109a8619fbfde44e40669R1116). But I can try using a CSS selector if that is preferrable.

<details>
<summary>Screen recordings</summary>

Before inline styles:

https://github.com/iTwin/iTwinUI/assets/45748283/8c0ecf58-34e2-40ff-8cc9-cf2922e650b8

After inline styles:

https://github.com/iTwin/iTwinUI/assets/45748283/0dc4df61-22f6-410e-bf94-de6f5dbd0b9c

</details>

Reply:
i think we should make a new selector `.iui-table-row-loading` or `.iui-table-row[data-iui-loading="true"]` that adds handles both sticky positioning and centering

Reply:
Sounds good, removed the two inlines in favor of `.iui-table-row[data-iui-loading="true"]`. (https://github.com/iTwin/iTwinUI/pull/1725/commits/29d1cb51469c267a866c28b9ecdb999802487409 and https://github.com/iTwin/iTwinUI/pull/1725/commits/b4403c0da9a216bb5b2b2e5218ade7f12ded6440)